cmd/minishift/cmd/config/config.go

```diff
@@ -59,6 +59,7 @@ var (
 	RemoteSSHUser         = createConfigSetting("remote-ssh-user", SetString, nil, nil, true, nil)
 	SSHKeyToConnectRemote = createConfigSetting("remote-ssh-key", SetString, nil, nil, true, nil)
 	TimeZone              = createConfigSetting("timezone", SetString, []setFn{validations.IsValidTimezone}, nil, true, nil)
+	RegistryLogin         = createConfigSetting("registry-login", SetBool, nil, nil, true, nil)
```

Comment:
the option is not clear by name what it does. it allows to login to ANY registry?

Reply:
IMO we should specifically say it is login for `registry.redhat.io`.  Then it will be easy for users to guess its purpose.

Reply:
I mean it should be named as `--redhat-registry-login` or something better.

Reply:
exactly my meaning... it is now left up for guesswork.